### DISCUSSION TITLE
Python with FZF is missing a venv selector

### DISCUSSION THREAD

**Discussion by Jomik** [LazyVim/LazyVim#4992] [Q&A] [answered]
We use linux-cultist/venv-selector.nvim for telescope
https://github.com/LazyVim/LazyVim/blob/8c79ab601af19b5370d560aa619567371ac70d86/lua/lazyvim/plugins/extras/lang/python.lua#L116
Is there a nice equivalent for fzf we can create, as I much prefer fzf?
Or any other recommendation? I suppose I could look into python automatically registering/activating common venv folders, but I am rather new with python, so I am unsure what is a sane setup and currently the manual selection works well.

**Comment by folke** [comment, marked as answer]
there's an open issue in the venv-selector repo

**Reply by Jomik** [reply]
Oh duh. Thanks 😂 
https://github.com/linux-cultist/venv-selector.nvim/issues/142

**Reply by ChenZhongPu** [reply]
So, `venv-selector` is disabled by default, right? I didn't set up any config for it.

<img width="401" height="113" alt="image" src="https://github.com/user-attachments/assets/198cf3bd-505a-4ac7-8ebd-e1ff9b1664f5" />

**Reply by Jomik** [reply]
No, it is because of 
```lua
    enabled = function()
      return LazyVim.has("telescope.nvim")
    end,
```
Which I think we need to remove now since the above PR has been merged.

**Reply by Jomik** [reply]
https://github.com/LazyVim/LazyVim/pull/5829 should resolve this :)
